values.yaml

```diff
@@ -1022,6 +1026,24 @@ ui:
     # @type: string
     annotations: null
 
+  # Configurations for displaying metrics in the UI.
+  metrics:
+    # Enable displaying metrics in the UI. The default value of "-"
+    # will inherit from global.metrics.enabled value.
+    # @type: boolean
+    # @default: global.metrics.enabled
+    enabled: "-"
+    # Provider for metrics. See
+    # https://www.consul.io/docs/agent/options#ui_config_metrics_provider
+    # This value is only used if ui.enabled is set to true.
+    # @type: string
+    provider: "prometheus"
+
+    # baseURL is the URL of the prometheus server, usually the service URL.
+    # This value is only used if ui.enabled is set to true.
+    # @type: string
+    baseURL: http://prometheus-server
```

Comment:
Should this be http://prometheus? The service name looks like it's just prometheus.

Reply:
Lurking on your PRs here: I think it'd be nice to keep it as prometheus-server so that it just works™️  with the prometheus community helm chart default values (https://github.com/prometheus-community/helm-charts/blob/c79e46a25fc4cff86deb9958be5b23c7a516ab2d/charts/prometheus/values.yaml#L602). If it doesn't make sense to make our templates do the same, then don't worry about it, but it'd be nice to have.

Reply:
DONE 😄